src/shared/lib/StoreUtils.spec.ts

```diff
@@ -288,10 +288,20 @@ describe('StoreUtils', () => {
         });
     });
 
-    it("won't fetch onkokb data if there are no mutations", (done) => {
-        fetchOncoKbData({}, emptyMutationData).then((data: any) => {
-            assert.deepEqual(data, {sampleToTumorMap: {}, indicatorMap: {}});
-            done();
+    describe('fetchOncoKbData', () => {
+        it("won't fetch onkokb data if there are no mutations", (done) => {
+            fetchOncoKbData({}, emptyMutationData).then((data: any) => {
```

Comment:
Actually, I copied this test some time ago while introducing `StoreUtils`. So, that `done` has always been there :)